code/parachain/frame/pablo/src/lib.rs

```diff
@@ -676,6 +676,7 @@ pub mod pallet {
 			// NOTE(connor): `start_block` must greater than current block
 			let start_block = frame_system::Pallet::<T>::current_block_number() + 1_u32.into();
 			let end_block = start_block + five_years_block.into();
+			let minimum_staking_amount: T::Balance = T::Convert::convert(10_000_u128);
```

Comment:
Can this be a documented const/passed through the config?

Reply:
All of the default poop values will be passed in somehow through the config, this is just a temporary value until the requirements are reviewed with product